### ISSUE TITLE
Zengarden improve ideas/some bugs

### ISSUE THREAD

**Issue by rogue-spectre** [supertuxkart/stk-code#2700]
Hi,

these two shortcuts seems unusable. With the one in the cavern we get at the end the barrier of the bridge. With the one passing through the little water area it's not possible to jump to the right on the road using nitro and it's not possible to go to the left using rocks because of kart stability problem.

![shortcut_improve1](https://cloud.githubusercontent.com/assets/2879782/21397822/77efee22-c7a5-11e6-8592-33c8b3559812.png)
![improve_shortcut2](https://cloud.githubusercontent.com/assets/2879782/21397835/84a30712-c7a5-11e6-89ec-4ca71471a492.png)

When can see booling float if released towards the mountain 
![2](https://cloud.githubusercontent.com/assets/2879782/21397395/ea3d2046-c7a3-11e6-9762-7e848d0e0de2.png)
![1](https://cloud.githubusercontent.com/assets/2879782/21397397/eb5d27f0-c7a3-11e6-81b0-e3ecf322e619.png)

There is a waterfall next to the bridge but there is no water in the canyon  
![rainfall](https://cloud.githubusercontent.com/assets/2879782/21397935/e0a9e30a-c7a5-11e6-896c-3b2b79dfbbc0.png)
![butnowater](https://cloud.githubusercontent.com/assets/2879782/21397938/e2b0885c-c7a5-11e6-8644-4920c85728c0.png)

The invisible walls around the temple are useless, since it would take the same time to use the road or to try the back of the temple + if one falls into the canyon, thunderbird will save us.

Nice holidays ;)

**Comment by auriamg** [comment]
1. The little caves around the bridge are not intended to be shortcuts, they're just scenery elements for explorers. That's not a bug.
2. As for the way on the pond, indeed it is unusable and the collectibles on it should just be removed
3. Bowling balls floating in the air : indeed, but very minor
4. No water in the canyon : very minor, might change if the track is remade but will not be a priority

**Comment by auriamg** [comment]
Issue 2 (shortcut on the pond) has been fixed in SVN (I removed the collectibles, so that the player doesn't believe they are intended to go there)

**Comment by samuncle** [comment]
yeah it was intended to be a shortcut but then I had other stuff to do and well I kinda forgot the map.

Anyway this map will be probably redone sooner or later and it will have a better layout and shortcuts that are easier to take

Sam

**Comment by qwertychouskie** [comment]
I fixed the water: [zenGarden.blend.zip](https://github.com/supertuxkart/stk-code/files/2107809/zenGarden.blend.zip)

**Comment by qwertychouskie** [comment]
@auriamg In that one I just added water where it was supposed to be, but I have another change coming soon that fixes other issues and replaces old textures with current-gen textures.

**Comment by auriamg** [comment]
@qwertychouskie Zengarden fix committed

**Comment by qwertychouskie** [comment]
I made a version with more fixes: https://jacobspctuneup.tk/STK/zengarden.zip

This replaces old textures with modern ones, and fixes a lot of small bugs.

**Comment by auriamg** [comment]
Your zip does not contain a license statement, we cannot used art without a license

**Comment by qwertychouskie** [comment]
Hmm, `stk_generic_rockSnow_a.png` came from the "textures" folder, but doesn't seem to have a documented license... @samuncle?  `stk_generic_rockSnow_b.png` is just a darkened version of `stk_generic_rockSnow_a.png` and is released under the same license, also same with `to_black_rockMountain.png`.  The original `rockTograss.png` and `blackRoad.png` are public-domain, so would inherit the license of `stk_generic_rockSnow_a.png`.

**Comment by qwertychouskie** [comment]
![screenshot_2018-08-11_22-38-09](https://user-images.githubusercontent.com/13989090/50367158-255f8a00-0533-11e9-8131-aa31dfb5b917.png)

Version with a proper `licenses.txt`:
https://jacobspctuneup.tk/STK/zengarden.zip

**Comment by auriamg** [comment]
@qwertychouskie just export and play. the vines are opaque because they use a jpg texture and not a png texture

**Comment by qwertychouskie** [comment]
Fixed, use https://jacobspctuneup.tk/STK/zengarden_fixvines.zip

**Comment by qwertychouskie** [comment]
Thanks!
